template/web/app/account-abstraction/index.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
```

Comment:
Also update this file: https://github.com/coinbase/build-onchain-apps/blob/main/src/create/experiences.ts

```typescript
export const experiences = [
  {
    value: 'buy-me-coffee',
    label: 'Buy Me A Coffee',
  }, {
    value: 'mint',
    label: 'Mint NFT',
  }, {
    value: 'account-abstraction',
    label: 'Account Abstraction',
  }
]
```

This will add `account-abstraction` as an option for cli create command

Reply:
https://github.com/coinbase/build-onchain-apps/tree/main/src#how-to-add-a-new-experience-option